utils/getFormatsDescription.test.ts

```diff
@@ -0,0 +1,176 @@
+import { expect, test } from "vitest";
+import getFormatsDescription from "./getFormatsDescription";
+
+// formats.length=1, qty=1, text, descriptions.length=1
+test("Render album description", () =>
+  expect(
+    getFormatsDescription([
+      {
+        name: "Vinyl",
+        qty: "1",
+        text: "Pink Marble",
+        descriptions: ["LP"],
+      },
+    ])
+  ).toStrictEqual("Pink Marble Vinyl LP"));
+
+// formats.length=1, qty>1, text, descriptions.length=1
+test("Render album description", () =>
+  expect(
+    getFormatsDescription([
+      {
+        name: "Vinyl",
+        qty: "2",
+        text: "Pink Marble",
+        descriptions: ["LP"],
+      },
+    ])
+  ).toStrictEqual("Pink Marble Vinyl LP (x2)"));
+
+// formats.length=1, qty=1, text, descriptions.length>1
+test("Get formats description", () =>
+  expect(
+    getFormatsDescription([
+      {
+        name: "Vinyl",
+        qty: "2",
+        text: "Pink Marble",
+        descriptions: ["LP"],
+      },
+    ])
+  ).toStrictEqual("Pink Marble Vinyl LP (x2)"));
+
+// formats.length=1, qty>1, text, descriptions.length>1
+test("Render album description", () =>
+  expect(
+    getFormatsDescription([
+      {
+        name: "Vinyl",
+        qty: "2",
+        text: "Pink Marble",
+        descriptions: ["LP", '12"'],
+      },
+    ])
+  ).toStrictEqual("Pink Marble Vinyl LP 12 (x2)"));
+
+// formats.length=1, qty=1, text, descriptions=undefined
+test("Get formats description", () =>
+  expect(
+    getFormatsDescription([
+      {
+        name: "Vinyl",
+        qty: "1",
+        text: "Pink Marble",
+      },
+    ])
+  ).toStrictEqual("Pink Marble Vinyl"));
+
+// formats.length=1, qty>1, text, descriptions=undefined
+test("Get formats description", () =>
+  expect(
+    getFormatsDescription([
+      {
+        name: "Vinyl",
+        qty: "2",
+        text: "Pink Marble",
+      },
+    ])
+  ).toStrictEqual("Pink Marble Vinyl (x2)"));
```

Comment:
you can just use `toBe` for these since `"Pink Marble Vinyl (x2)" === "Pink Marble Vinyl (x2)"`

```suggestion
  ).toBe("Pink Marble Vinyl (x2)"));
```